src/components/home/Hero.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, Text, Flex } from "@radix-ui/themes";
+import HeroSection from "../ui/HeroSection";
+import { FaRegCircleCheck } from "react-icons/fa6";
+
+const Hero = () => {
+  return (
+    <>
+      <Box
+        style={{
+          backgroundColor: "#5AC597",
+        }}
+      >
+        <Flex p="3" align="center" gap="2" mx="2">
+          <FaRegCircleCheck />
+          <Text>Join our team of open source tech volunteers!</Text>
+        </Flex>
+      </Box>
+      <HeroSection
+        heading="Your Mutual Aid Logistic
+        Experts"
+        imgSrc="/images/home/hero-image.jpg"
+        imgAlt=""
+        buttonHeading="Ship aid to people in need."
+        buttons={[
+          {
+            label: "Donate with PayPal",
+            href: "",
```

Comment:
I assume we have an eye on all the links on the website that still need a place to go to? If not we should probably have a general ticket for "check links" or something?